### ISSUE TITLE
Tutorial for mapping of new data

### ISSUE THREAD

**Issue by emdann** [Center-for-Health-Data-Science/multiDGD#6]
So far there's a demo for training on test set, but not for using a trained model on new data.

**Comment by viktoriaschuster** [comment]
So far we have training (which maps a validation set automatically) and then mapping of the held out test set to latent space. For using completely new data, we would need data with the same called regions in the ATAC data (I guess).

What do you think is a good real-world example we could add as a tutorial?

**Comment by emdann** [comment]
Thanks @viktoriaschuster, for now I was thinking mostly of application to mapping scRNA-seq data to predict chromatin profiles or allow joint analysis. I think a good tutorial example could be using the mouse gastrulation model with the scRNA-seq data from [Pijuan-Sala et al. 2019](https://www.nature.com/articles/s41586-019-0933-9) (which is available in anndata format from [scvelo](https://scvelo.readthedocs.io/en/stable/scvelo.datasets.gastrulation/#scvelo.datasets.gastrulation))

**Comment by viktoriaschuster** [comment]
Good idea. I am working on a tutorial with our mouse_gastrulation model now and will solve this issue and the single-prediction issue together.
